src/Watcher/watch_log.py

```diff
@@ -30,10 +32,10 @@ def update_csv(date, Data):
 # also if usr is AFK then append line
 def import_data(file):
     with open(file, 'r') as f:
```

Comment:
add error handling here.
FileNotFound error occurs if user installed the watcher and run watcher -s command. 
![image](https://user-images.githubusercontent.com/86405648/196662898-18a98bc8-ffb0-494d-88d5-aa8caebf835e.png)